jobserver/models/core.py

```diff
@@ -939,11 +967,14 @@ def get_full_name(self):
         """Support Django's User contract"""
         return self.fullname
 
-    def get_password_reset_url(self):
-        uid = urlsafe_base64_encode(force_bytes(self.pk))
-        token = PasswordResetTokenGenerator().make_token(self)
+    def get_login_url(self, max_age=timedelta(hours=1)):
+        token = new_ulid_str()
```

Comment:
I don't think ulid is the best choice for a secret token here, as it's partially predictable by design. 

`secrets.token_urlsafe(64)` is a better option, I think?

Reply:
I always forget about the secrets module 👌